readme.md

```diff
@@ -14,6 +14,8 @@ As a computer expert, we want to be more efficient and do our jobs better. We kn
 
 ## Quick links
 
+1. [find a stale file](#findFile)
```

Comment:
Will do, I was just about to ask, I added them at the top just for us to see them quicker. 

I agree, find a file should go directly after list folder contents